spec/fixtures/iex/balance_sheet/msft.yml

```diff
@@ -0,0 +1,53 @@
+---
+http_interactions:
```

Comment:
I ended up copying the majority of these fixture files from other examples but based o n the various dates throughout the files it seems as though a generator of sorts was used. Please advise if I should do things differently than I have here.

Reply:
These are not using a generator but recorded using [vcr](https://github.com/vcr/vcr). Delete the .yml files you copied, set the tokens in your ENV and run `rspec ...spec.rb` without the presence of the VCR file and it will record new interactions.

Reply:
Cool thanks for the info. I currently don't have a paid IEX account so I don't believe this will work for me since balance sheet is a premium endpoint. This seems like a common issue I imagine you have come across in the past, is there a good way around this?

Reply:
i did, no way around this, i just pay for it - maybe email them and they can give you some credits to contribute here?

Reply:
I'll just pay for the account i was planning to eventually anyway. Will look into doing this properly this weekend.

Reply:
Updated with proper fixture files.